docs/HowToFAQ/Errors-Resolution.md

```diff
@@ -519,15 +573,67 @@ considered ghost. And then the statement can contain no substatements that are f
 `return` and `yield` stastements are never ghost, so they cannot appear in a statement whose guarding
 value is ghost.
 
-## **cannot assign to _var_ in a ghost context**
+## **Error: cannot assign to _var_ in a ghost context** {#r_no_assign_to_var_in_ghost}
 
-<!-- TODO -->
+```dafny
+method m(ghost c: int) 
+{
+  var x := 7;
+  if (c == 1) { x :| x < 8; }
+}
+```
 
-## **_var_ cannot be assigned a value that depends on a ghost**
+No changes to non-ghost variables may occur in ghost contexts.
+Ghost context can be implicit, such as the branches of an if-statement
+whose condition is a ghost expression.
 
-<!-- TODO -->
+## **Error: non-ghost _var_ cannot be assigned a value that depends on a ghost** {#r_no_assign_ghost_to_var}
```

Comment:
I find "a value that depends on a ghost" confusing, as it implies value have lingering dependencies on other values, which doesn't make sense to me in a language without lazy evaluation semantics. Would it be accurate to just say "a ghost value"?